src/terra-dev-site/test/application-status-overlay/StatusOverlayAllProps.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ApplicationStatusOverlay from '../../../../lib/application-status-overlay/ApplicationStatusOverlay';
+import ApplicationStatusOverlayProvider from '../../../../lib/application-status-overlay/ApplicationStatusOverlayProvider';
+
+const ApplicationStatusOverlayAllProps = () => {
+  const StatusViewButtons = [
+    {
+      text: 'Action 1',
+      key: 1,
+      size: 'medium',
+    }, {
+      text: 'Action 2',
+      key: 2,
+      size: 'medium',
+    },
+  ];
+
+  return (
+    <ApplicationStatusOverlayProvider>
+      <ApplicationStatusOverlay buttonAttrs={StatusViewButtons} message="Status View with all props" title="All props test" variant="no-data" />
```

Comment:
with `title` prop removed, it will need to be removed here as well:
```suggestion
      <ApplicationStatusOverlay buttonAttrs={StatusViewButtons} message="Status View with all props" variant="no-data" />
```

Reply:
Addressed here : https://github.com/cerner/terra-application/pull/47/commits/6fca8b18fef599c004d9efd588517068495dd45f